pkg/cmd/cli/cmd/importimage.go

```diff
@@ -98,8 +99,14 @@ func RunImportImage(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, arg
 
 	if stream.Annotations != nil {
 		delete(stream.Annotations, imageapi.DockerImageRepositoryCheckAnnotation)
+	} else {
+		annotations := make(map[string]string)
```

Comment:
One last nit, sorry I didn't spotted that eariler, merge the two lines into one. There's no need for new variable here.